api/app/signals/apps/api/v1/validation/address/pdok.py

```diff
@@ -28,7 +34,7 @@ def _search_result_to_address(self, result):
             sia_address_dict[sia_key] = result[PDOK_key] if PDOK_key in result else ''
         return sia_address_dict
 
-    def _pdok_request_query_params(self, address):
+    def _pdok_request_query_params(self, address, lon=None, lat=None):
```

Comment:
Small remark:

I have had problems debugging if in a chain of function calls each one defines default values. Since this is all internal I would set the defaults at the boundary with public methods/function calls and pass the data on from there.